apko-snapshot/action.yaml

```diff
@@ -74,12 +80,13 @@ runs:
 
     # Only publish the versioned tag to start.  After we have signed and
     # attested things, then we use crane to update :latest below.
-    - uses: chainguard-dev/actions/apko-build@main
+    - uses: jdolitsky/actions/apko-build@use-oci-mediatypes-testing
```

Comment:
Should be reverted once this PR is merged: https://github.com/chainguard-dev/actions/pull/85/files